src/Tutor.Core/DomainModel/Feedback/FeedbackService.cs

```diff
@@ -0,0 +1,26 @@
+﻿using FluentResults;
+using System.Collections.Generic;
+
+namespace Tutor.Core.DomainModel.Feedback
+{
+    public class FeedbackService : IFeedbackService
+    {
+        private readonly IFeedbackRepository _ifeedbackRepository;
+
+        public FeedbackService(IFeedbackRepository ifeedbackRepository)
+        {
+            _ifeedbackRepository = ifeedbackRepository;
+        }
+
+        public Result<EmotionsFeedback> SaveEmotionsFeedback(EmotionsFeedback emotionsFeedback)
+        {
+            if (emotionsFeedback == null)
```

Comment:
For fail fast/return early we follow the convention of one line IF-s. I.e.:

if(emotionsFeedback == null) return Result.Fail("Empty Emotions Feedback");